lib/static/new-ui/features/suites/components/TestNameFilter/index.tsx

```diff
@@ -27,10 +32,49 @@ function TestNameFilterInternal(props: TestNameFilterProps): ReactNode {
 
     const isInitialized = useSelector(getIsInitialized);
 
-    return <TextInput disabled={!isInitialized} placeholder='Search or filter' value={testNameFilter} onChange={onChange}/>;
+    const onCaseSensitiveClick = ():void => {
```

Comment:
nit: weird formatting... usually it's `(): void` (would be nice to have it automated, but that's complicated, we didn't come to an agreement on this so far.)